src/js/Rickshaw.Graph.Band.js

```diff
@@ -0,0 +1,40 @@
+Rickshaw.namespace("Rickshaw.Graph.Band");
```

Comment:
should this be namespaced under Rickshaw.Graph.Render?

Reply:
As far as I can tell the renders are all line renderers, they're loaded into an object and can be used based on how you want to render plotted lines. 

I originally thought about putting this under Render, but Band rendering doesn't need any of the underlying utility functions that are provided by the parent class. 

Renderers also assume they're the only thing rendering during a re-paint and completely clear the graph. So just adding two renderers to the same structure naively would only paint the most recently rendered data (band or lines).